manala.influxdb/defaults/main.yml

```diff
@@ -5,6 +5,14 @@ manala_influxdb_install_packages: ~
 manala_influxdb_install_packages_default:
   - influxdb
 
+# Dir
+manala_influxdb_meta_dir: ~
```

Comment:
What do you thing to regroup all dir variables under a `manala_influxdb_directories` list ? 

We should use only one file task : 
```
- name: dir > Create directory
  file:
    path: "{{ item }}"
    state: directory
    owner: "{{ manala_influxdb_directories_user }}"
    group: "{{ manala_influxdb_directories_group }}"
    mode: "{{ manala_influxdb_directories_mode }}"
  with_items: "{{ manala_influxdb_directories }}"
```

Reply:
Done. It is cleaner indeed !

Reply:
although we can't do that with only one variable :

```
    manala_influxdb_config:
      - reporting-disabled: true
      - meta:
        - dir: "{{ manala_influxdb_meta_dir }}"
      - data:
        - dir: "{{ manala_influxdb_data_dir }}"
        - wal-dir: "{{ manala_influxdb_wal_dir }}" 
```

Reply:
IMHO you should not reference directly role variables in your playbook ; instead you should do something like that : 

```
_influxdb_dirs:
 data: /mnt/influxdb/data
 wal: /mnt/influxdb/wal

manala_influxdb_directories: "{{ _influxdb_dirs.values() | list }}"
manala_influxdb_config:
  - meta: 
    - dir: "{{ _influxdb_dirs.dir }}"
  - data:
    - dir: "{{ _influxdb_dirs.data }}"
    - wal-dir: "{{ _influxdb_dirs.wal_dir }}"
```